MdeModulePkg/Include/Guid/SmBiosTableHob.h

```diff
@@ -0,0 +1,20 @@
+/** @file
+ Define the GUID gPldSmbios3TableGuid and gPldSmbiosTableGuid HOB struct.
+
+Copyright (c) 2021, Intel Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _SMBIOS_TABLE_HOB_H_
+#define _SMBIOS_TABLE_HOB_H_
+
+#include <Uefi.h>
+
+typedef struct {
+  EFI_PHYSICAL_ADDRESS          SmBiosTableAddress;

```

Comment:
I understand Laszlo proposed to use EFI_PHYSICAL_ADDRESS other than UINT64. But using the EFI_XXX type requires dependency on UEFI types.

@mauricema , @gdong1 , @vincent-j-zimmer , what's your thought?